src/utils/frontend-api-client/appointments.js

```diff
@@ -0,0 +1,16 @@
+import axios from 'axios'
+
+export const getAvailableAppointments = async (
+  workOrderReference,
+  fromDate,
+  toDate
+) => {
+  const params = new URLSearchParams()
+  params.append('workOrderReference', workOrderReference)
+  params.append('fromDate', fromDate.toISOString().substring(0, 10))
+  params.append('toDate', toDate.toISOString().substring(0, 10))
+
+  const { data } = await axios.get(`/api/appointments?${params.toString()}`)
```

Comment:
An alternative syntax for building the query params would be this, which I think we've used in similar functions:

```
axios.get('/api/appointments', {
    params: {
      workOrderReference,
      fromDate: fromDate.toISOString().substring(0, 10),
      toDate: toDate.toISOString().substring(0, 10)
    }
  })
```